store/store.go

```diff
@@ -67,6 +67,21 @@ type Store struct {
 	stateChangedNotifee *stateChangedNotifee
 }
 
+func defaultIndexFunc(s *Store) func(model string, key ds.Key, oldData, newData []byte, txn ds.Txn) error {
```

Comment:
The only one for now, but in theory, other indexing functions could be used/implemented.